lawnchair/src/app/lawnchair/ui/preferences/destinations/QuickstepPreferences.kt

```diff
@@ -36,17 +37,26 @@ fun QuickstepPreferences(
         context.packageManager.getLaunchIntentForPackage("com.google.ar.lens") != null
     }
 
-    if (!LawnchairApp.isRecentsEnabled) QuickSwitchIgnoredWarning()
-
     PreferenceLayout(
         label = stringResource(id = R.string.quickstep_label),
         modifier = modifier,
     ) {
+        if (!LawnchairApp.isRecentsEnabled) QuickSwitchIgnoredWarning()
         PreferenceGroup(heading = stringResource(id = R.string.general_label)) {
             SwitchPreference(
                 adapter = prefs.recentsTranslucentBackground.getAdapter(),
                 label = stringResource(id = R.string.translucent_background),
             )
+            val recentsTranslucentBackground by prefs.recentsTranslucentBackground.observeAsState()
+            ExpandAndShrink(visible = recentsTranslucentBackground) {
+                SliderPreference(
+                    adapter = prefs.recentsTranslucentBackgroundAlpha.getAdapter(),
+                    label = stringResource(id = R.string.translucent_background_alpha),
+                    step = 0f,
```

Comment:
```suggestion
                    step = 0.05f,
```
The step here is too broad. I'd recommend adding some constraints, like `5%` step.